TYPO3.Flow/Classes/TYPO3/Flow/Command/PackageCommandController.php

```diff
@@ -63,21 +64,21 @@ public function injectSettings(array $settings)
      * @return string
      * @see typo3.kickstart:kickstart:package
      */
-    public function createCommand($packageKey, $packageType = 'typo3-flow-package')
+    public function createCommand($packageKey, $packageType = PackageInterface::DEFAULT_COMPOSER_TYPE)
     {
         if (!$this->packageManager->isPackageKeyValid($packageKey)) {
-            $this->outputLine('The package key "%s" is not valid.', array($packageKey));
+            $this->outputLine('The package key "%s" is not valid.', [$packageKey]);
             $this->quit(1);
         }
         if ($this->packageManager->isPackageAvailable($packageKey)) {
-            $this->outputLine('The package "%s" already exists.', array($packageKey));
+            $this->outputLine('The package "%s" already exists.', [$packageKey]);
             $this->quit(1);
         }
-        if (substr($packageType, 0, 11) !== 'typo3-flow-') {
-            $this->outputLine('The package must be a Flow package, but "%s" is not a valid Flow package type.', array($packageKey));
+        if (ComposerUtility::isFlowPackageType($packageType)) {
```

Comment:
Can we make 'isFlowPackageType' a  bit more clear? In CR we use NodeType->isOfType, something similar here ends up in a long name so I don't really have an awesome suggestion coming up, but maybe someone has a brilliant idea? Otherwise leave it as is ;)